backend/supabase/seed.sql

```diff
@@ -110,7 +110,7 @@ create table if not exists
     user_id text not null,
     contract_id text not null,
     data jsonb not null,
-    fs_updated_time timestamp not null,
+    fs_updated_time timestamp,
```

Comment:
Why change this to nullable?

Reply:
as it is not copied in via the replicator, it will not have an fs_updated_time (and actually the column can be dropped)